template/files/circleci_config.yml

```diff
@@ -0,0 +1,69 @@
+# Ruby CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-ruby/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+      - image: circleci/ruby:2.3-node
```

Comment:
Might want to replace the image with:
`image: circleci/ruby:2.3-stretch-node`

Not completely sure it's still an issue but there was a circleci cache issue we were having that required us to re-trigger some builds as they'd fail on the first run: https://support.circleci.com/hc/en-us/articles/360004937834-Resolving-Ruby-Jessie-to-Stretch-issue-cannot-open-shared-object-file-cache-problem

Reply:
Will do so

Reply:
done